data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/SummaryDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.bottlerocketstudios.brarchitecture.data.model
+
+import android.os.Parcelable
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kotlinx.parcelize.Parcelize
+
+@JsonClass(generateAdapter = true)
+@Parcelize
+data class SummaryDto(
+    @Json(name = "type") val type: String?,
+    @Json(name = "raw") val raw: String?,
+    @Json(name = "markup") val markUp: String?,
+    @Json(name = "html") val created_on: String?,
```

Comment:
@Br-DiorD Is this property html OR creation date? It seems like one of the values need to be updated to match the other based on the actual backend api response.

Reply:
@br-jared-stockton typo